.github/workflows/test_with_woodwork_main_branch.yml

```diff
@@ -0,0 +1,61 @@
+name: Test Featuretools with main Branch
+on:
+  schedule:
+    - cron: "0 * * * *"
+  pull_request:
+    types: [opened, synchronize]
+  push:
+    branches:
+      - main
+  workflow_dispatch:
+jobs:
+  scheduled_unit_latest_tests:
+    name: ${{ matrix.python_version }} unit tests ${{ matrix.libraries }}
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: true
+      matrix:
+        python_version: ["3.8", "3.9", "3.10"]
```

Comment:
Featuretools doesn't support `3.10` yet, but we still do support `3.7`, so It think this list should be `["3.7", "3.8", "3.9"]` instead.

Reply:
updated